### ISSUE TITLE
install error FileNotFoundError: [Errno 2] No such file or directory: 'tests/requirements.txt'

### ISSUE THREAD

**Issue by mouadhkaabachi** [singingwolfboy/flask-dance#319]
image: python:3.7-stretch

--------------------------------------------------------------------------------
Collecting flask-dance==3.0.0
  Downloading Flask-Dance-3.0.0.tar.gz (1.4 MB)
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-3yf_59su/flask-dance/setup.py'"'"'; __file__='"'"'/tmp/pip-install-3yf_59su/flask-dance/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-ff3727ut
         cwd: /tmp/pip-install-3yf_59su/flask-dance/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-3yf_59su/flask-dance/setup.py", line 63, in <module>
        tests_require=get_requirements("tests/requirements.txt"),
      File "/tmp/pip-install-3yf_59su/flask-dance/setup.py", line 38, in get_requirements
        with open(path) as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'tests/requirements.txt'
    ----------------------------------------

**Comment by ilyakamens** [comment]
I seem to be encounter this as well.

`Dockerfile`:
```Dockerfile
FROM python:3.7-alpine3.11 AS base

FROM base AS builder

RUN apk add --no-cache gcc git musl-dev postgresql-dev \
    # For PyNaCl
    libffi-dev make
COPY requirements.txt /requirements.txt
RUN mkdir /install
RUN pip install pip-accel==0.43
RUN pip-accel install \
    --install-option="--prefix=/install" \
    --install-option="--no-warn-script-location" \
    -r /requirements.txt \
    awscli
```

`requirements.txt`:
```
#
# This file is autogenerated by pip-compile
# To update, run:
#
#    fcs update-requirements
#
amqp==2.6.1               # via kombu
appdirs==1.4.4            # via black
attrs==20.2.0             # via black, jsonschema, pytest, pytest-recording
backcall==0.2.0           # via ipython
bcrypt==3.2.0             # via paramiko
beautifulsoup4==4.8.2     # via -r requirements.in
billiard==3.6.3.0         # via celery
black==19.10b0            # via -r requirements.in
blinker==1.4              # via sentry-sdk
cachetools==4.1.1         # via zenpy
celery==4.4.5             # via -r requirements.in, flower
certifi==2020.6.20        # via requests, sentry-sdk
cffi==1.14.2              # via bcrypt, cryptography, pynacl
chardet==3.0.4            # via requests
click==7.1.2              # via black, flask, pip-tools
colorama==0.4.3           # via pytest-watch
coverage==5.2.1           # via pytest-cov
cryptography==3.1         # via paramiko
decorator==4.4.2          # via ipython
docopt==0.6.2             # via pytest-watch
entrypoints==0.3          # via flake8
flake8-commas==2.0.0      # via -r requirements.in
flake8-docstrings==1.5.0  # via -r requirements.in
flake8-isort==2.9.1       # via -r requirements.in
flake8-quotes==2.1.0      # via -r requirements.in
flake8==3.7.9             # via -r requirements.in, flake8-commas, flake8-docstrings, flake8-isort, flake8-quotes
flask-admin==1.5.6        # via -r requirements.in
flask-dance==3.0.0        # via -r requirements.in
```

Stacktrace:
```
...
2020-11-04 21:24:08 71761796d6b5 pip.req.req_set[1] INFO Collecting flask-dance==3.0.0 (from -r /requirements.txt (line 28))
2020-11-04 21:24:08 71761796d6b5 pip.download[1] INFO File was already downloaded /var/cache/pip-accel/sources/Flask-Dance-3.0.0.tar.gz
2020-11-04 21:24:08 71761796d6b5 pip.utils[1] INFO Complete output from command python setup.py egg_info:
2020-11-04 21:24:08 71761796d6b5 pip.utils[1] INFO Traceback (most recent call last):
  File "<string>", line 20, in <module>
  File "/tmp/pip-accel-build-dir-_968nmob/flask-dance/setup.py", line 63, in <module>
    tests_require=get_requirements("tests/requirements.txt"),
  File "/tmp/pip-accel-build-dir-_968nmob/flask-dance/setup.py", line 38, in get_requirements
    with open(path) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/requirements.txt'

----------------------------------------
2020-11-04 21:24:08 71761796d6b5 pip_accel.cli[1] ERROR Caught unhandled exception!
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/pip_accel/cli.py", line 57, in main
    accelerator.install_from_arguments(arguments)
  File "/usr/local/lib/python3.7/site-packages/pip_accel/__init__.py", line 237, in install_from_arguments
    requirements = self.get_requirements(arguments, use_wheels=self.arguments_allow_wheels(arguments))
  File "/usr/local/lib/python3.7/site-packages/pip_accel/__init__.py", line 310, in get_requirements
    return self.download_source_dists(arguments, use_wheels=use_wheels)
  File "/usr/local/lib/python3.7/site-packages/pip_accel/__init__.py", line 410, in download_source_dists
    requirements = self.get_pip_requirement_set(arguments, use_remote_index=True, use_wheels=use_wheels)
  File "/usr/local/lib/python3.7/site-packages/pip_accel/__init__.py", line 497, in get_pip_requirement_set
    requirement_set = command.run(opts, args)
  File "/usr/local/lib/python3.7/site-packages/pip/commands/install.py", line 294, in run
    requirement_set.prepare_files(finder)
  File "/usr/local/lib/python3.7/site-packages/pip/req/req_set.py", line 334, in prepare_files
    functools.partial(self._prepare_file, finder))
  File "/usr/local/lib/python3.7/site-packages/pip/req/req_set.py", line 321, in _walk_req_to_install
    more_reqs = handler(req_to_install)
  File "/usr/local/lib/python3.7/site-packages/pip/req/req_set.py", line 505, in _prepare_file
    abstract_dist.prep_for_dist()
  File "/usr/local/lib/python3.7/site-packages/pip/req/req_set.py", line 123, in prep_for_dist
    self.req_to_install.run_egg_info()
  File "/usr/local/lib/python3.7/site-packages/pip_accel/__init__.py", line 728, in run_egg_info_wrapper
    return original_method(self, *args, **kw)
  File "/usr/local/lib/python3.7/site-packages/pip/req/req_install.py", line 410, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/usr/local/lib/python3.7/site-packages/pip/utils/__init__.py", line 733, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip.exceptions.InstallationError: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-accel-build-dir-_968nmob/flask-dance
ERROR: Service 'fccelery_worker' failed to build : The command '/bin/sh -c pip-accel install     --install-option="--prefix=/install"     --install-option="--no-warn-script-location"     -r /requirements.txt     awscli' returned a non-zero code: 1
```

Any help here would be much appreciated @singingwolfboy!

**Comment by singingwolfboy** [comment]
This issue should be resolved as of Flask-Dance 3.1.0.

**Comment by ilyakamens** [comment]
> This issue should be resolved as of Flask-Dance 3.1.0.

Thanks!
